verifier/verifier.go

```diff
@@ -177,6 +179,18 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 			return err
 		}
 
+		pluginVersion := metadata.Version
+		var semVerRegEx = regexp.MustCompile(`^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`)
+
+		//checking if the plugin version is in valid semver format
+		if !semVerRegEx.MatchString(pluginVersion) {
```

Comment:
What happens when we pass empty value.

Reply:
It does not throw any error, the execution will go inside the if condition though